galaxy_ng/app/settings.py

```diff
@@ -115,6 +116,23 @@
 # https://django-guardian.readthedocs.io/en/stable/configuration.html#anonymous-user-name
 ANONYMOUS_USER_NAME = None
 
+# L10N settings
+USE_L10N = True
+USE_I18N = True
+
+# Default language
+LANGUAGE_CODE = 'en-us'
+
+LANGUAGES = [
+    ('ja', 'Japanese'),
+    ('nl', 'Dutch'),
+    ('fr', 'French'),
+    ('es', 'Spanish'),
+    ('zh-hans', 'Chinese'),
+]
+
+LOCALE_PATHS = (os.path.join('/src/galaxy_ng/locale'), )
```

Comment:
@jerabekjiri 

I think you need to move the `locale` folder from the root of repo to inside the `galaxy_ng` module folder.

`mv locale galaxy_ng/locale`

then this setting can be

```suggestion
from pathlib import Path  # move this line to the top of the file
BASE_DIR = Path(__file__).absolute().parent
LOCALE_PATHS = (os.path.join(BASE_DIR, 'locale'),)
```

Reply:
Here an example https://github.com/ansible/awx/tree/devel/awx/locale and https://github.com/ansible/awx/blob/9da383fe2d3bd803fd6e6766246e3b4f6566bd96/awx/settings/defaults.py#L140